buildSrc/settings.gradle.kts

```diff
@@ -0,0 +1,32 @@
+dependencyResolutionManagement {
+    versionCatalogs {
+        create("libs") {
+            from(files("../gradle/libs.versions.toml"))
+
+            val kotlinVersion = providers.gradleProperty("kotlin_version").orNull
```

Comment:
I don't like that we have 3 ways of overriding Kotliin version. So I'm planning to discuss it with the author, and maybe after some changes in our infrastructure build scripts, I'll leave only one.

Reply:
Good idea - can you make an issue to keep track, and add a `TODO <link to issue>` in the code?

It'd also be a good idea to de-duplicate the version override in the actual project and in buildSrc.

Reply:
added https://github.com/Kotlin/kotlinx-atomicfu/issues/421